connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSourceTaskTest.java

```diff
@@ -220,17 +224,22 @@ public void testMissingFile() throws InterruptedException {
         task.stop();
     }
 
+    @Test
     public void testInvalidFile() throws InterruptedException {
         config.put(FileStreamSourceConnector.FILE_CONFIG, "bogusfilename");
         task.start(config);
         // Currently the task retries indefinitely if the file isn't found, but shouldn't return any data.
-        for (int i = 0; i < 100; i++)
+        for (int i = 0; i < 3; i++)
             assertNull(task.poll());
     }
 
-
     private void expectOffsetLookupReturnNone() {
-        EasyMock.expect(context.offsetStorageReader()).andReturn(offsetStorageReader);
-        EasyMock.expect(offsetStorageReader.offset(EasyMock.<Map<String, String>>anyObject())).andReturn(null);
+        when(context.offsetStorageReader()).thenReturn(offsetStorageReader);
+        when(offsetStorageReader.offset(anyMap())).thenReturn(null);
+    }
+
+    private void verifyAll() {
```

Comment:
Most tests end up calling this method twice, once explicitly and once via `teardown()`. Let's pick one way and stick with it.

Reply:
On I forgot about this, I removed the extra call after each method.